usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -404,3 +405,12 @@ def fetch_state_name_from_code(state_code):
         logger.warning("{} not found for state_code: {}".format(",".join(columns), state_code))
         return None
     return result[columns[0]]
+
+
+def fetch_naics_description_from_code(naics_code, passthrough=None):
```

Comment:
The function here is architected more flexibly than the name implies... i.e., you could add additional attributes to the "column" dict, and be fetching more than "description", so it might make sense to name the function something like fetch_naics_attributes_from_code and also potentially pass in the "columns" dict as an arg... not something that has to be changed, just a thought.

Reply:
I made from a copy of other similar functions in this file. The `columns` list is more useful for lines 414 and 416 than for 412 (if that makes sense).
I agree it could be more capable and it wouldn't require much effort, but there isn't any need.